HISTORY.md

```diff
@@ -0,0 +1,1482 @@
+---
+title: Release History
+---
+
+# 2.1.4 (2023-11-7)
+
+This release updates mesa-viz-tornado dependency v0.1.3. This removes
+the external JavaScript templates and
+`prevents 404 errors https://github.com/projectmesa/mesa-viz-tornado/issues/40`\_
```

Comment:
This link set up doesn't seem to convert in MarkDown... needs to be [prevents 404 errors](https://github.com/projectmesa/mesa-viz-tornado/issues/40)